dev-infrastructure/templates/svc-cluster.bicep

```diff
@@ -76,6 +76,9 @@ param aksEtcdKVEnableSoftDelete bool = true
 @description('IPTags to be set on the cluster outbound IP address in the format of ipTagType:tag,ipTagType:tag')
 param aksClusterOutboundIPAddressIPTags string = ''
 
+@description('Enable Swift V2 for the AKS cluster and VNET')
+param aksEnableSwift bool
```

Comment:
do you want to default this to false for svc cluster, or require it as a param?  I'm good either way just wondering since svc clusters probably wont have it enabled.

Reply:
you are right. i removed the option for the SVC cluster and defaulted to false